cmd/dra-example-kubeletplugin/state.go

```diff
@@ -222,8 +232,11 @@ func (s *DeviceState) syncPreparedDevicesToCRDSpec(spec *nascrd.NodeAllocationSt
 				}
 				prepared.Gpu.Devices = append(prepared.Gpu.Devices, outdevice)
 			}
+		default:
+			return fmt.Errorf("wrong prepared device type: %v", devices.Type())
```

Comment:
Can we make this the following to be consistent with line 115:
```
		return fmt.Errorf("unknown device type: %v", devices.Type())
```

Reply:
Done.